check_run/public/js/check_run/CheckRun.vue

```diff
@@ -216,6 +217,19 @@ function update_sort(key_name) {
 }
 
 function onMOPChange(event, rowName) {
+	// change of mode of payment make document Unsaved
+	//start
+	var element1 = document.querySelector(`#${rowName}`)
+	element1.checked = !element1.checked
+	console.log(element1.checked)
+	var changeEvent1 = new Event('change');
+	element1.dispatchEvent(changeEvent1);
+	var element2 = document.querySelector(`#${rowName}`)
+	element2.checked = !element1.checked
+	var changeEvent2 = new Event('change');
```

Comment:
We want to be idiomatic about using Vue or using the Frappe form API, follwing the conventions set by Frappe (the comapny). This vanilla JS approach, while valid and normally considered to be the most universally readable approach, isn't either of those.

Reply:
`frm.dirty()`